fastly.yaml

```diff
@@ -0,0 +1,4 @@
+---
+vhost:
```

Comment:
But, you're right re: this example file. I'll put the current content in comments and provide a more specific example below.